code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -304,25 +302,17 @@ Please contact me on #coderbus IRC. ~Carnie x
 
 
 /mob/living/carbon/human/update_inv_head()
-	var/obj/item/H = ..()
-	if(H)
-		if(client && hud_used && hud_used.hud_shown)
-			if(hud_used.inventory_shown)				//if the inventory is open ...
-				H.screen_loc = ui_head		//TODO	//...draw the item in the inventory screen
-			client.screen += H						//Either way, add the item to the HUD
-

```

Comment:
human and monkey now both use the same carbon/update_inv_head(), and the relevant hud updates are done by calling proc/update_hud_head()